GVRf/Sample/simple-sample/src/org/gearvrf/simplesample/SampleViewManager.java

```diff
@@ -36,34 +57,194 @@ public void onInit(GVRContext gvrContext) {
+		// If you want to use specific MediaPlayer, you can make your own object like GVRVideoObject.
+		
+		return videoObject;
+	}
+	
+	
+	
+	// 3. Create webview object
+	private GVRSceneObject buildWebViewObject(){
+
+		// Create your own WebView.
+		// This WebView follow Android WebView.
+		WebView webView = new WebView(mGVRContext.getContext());
+		webView.setInitialScale(100);
+		webView.measure(2000, 1000);
+		webView.layout(0, 0, 2000, 1000);
+		webView.setWebViewClient(new WebViewClient());
+		webView.loadUrl("http://www.google.com");
+		
+		WebSettings set = webView.getSettings();
+		set.setJavaScriptEnabled(true);
+		
+		
+		// From inside of GVRWebViewSceneObject, refresh per 30 fps. ( 0.5 sec ). Also you can change from there.
+		GVRWebViewSceneObject webviewObject = new GVRWebViewSceneObject(mGVRContext, 4.0f, 2.0f, webView);
+		
+		return webviewObject;
+	}
+	
+	
+	
+	// 4. Create camera object
+	private GVRSceneObject buildCameraObject(){
+		
+		Camera camera = Camera.open();
+		
+		GVRCameraSceneObject cameraObject = new GVRCameraSceneObject(mGVRContext, 4.0f,  2.0f, camera);
+		
+		return cameraObject;
+	}
+	
+	
+	
+	// 5. Create model object
+	private GVRSceneObject buildModelObject(){
+
+		// Load mesh from GVRAndroidResouce.
+		// Mesh load use Assimp library. 
+		// Please see "http://assimp.sourceforge.net"
+        GVRMesh sphereMesh = mGVRContext.loadMesh(new GVRAndroidResource(mGVRContext, R.raw.sphere));
+
+        GVRSceneObject modelObject = new GVRSceneObject(mGVRContext, sphereMesh,
+        		mGVRContext.loadTexture(new GVRAndroidResource(mGVRContext, R.drawable.earthmap1k)));
+        
+        return modelObject;
+	}
+	
+	
+	
+	// 6. Create animation
+	private void buildAnimation(){
+		
+		// Create rotation animation.
+		GVRAnimation rotAnimation = new GVRRotationByAxisAnimation(	//
+											mMainObject, 50f, 360.0f, //
+											0.0f, 1.0f, 0.0f);
+		// If you set repeat count to -1, it will be infinite.
+		rotAnimation.setRepeatMode(GVRRepeatMode.REPEATED).setRepeatCount(-1);
+        
+		
+        rotAnimation.start(mGVRContext.getAnimationEngine());
+	}
+	
+	
+	
+	// 7. Build eye point holder for picking.
+	private void buildEyePointeeHolder(){
+		
+		// Create eye point holder.
+		// This holder will hold object's eye point.
+		// And eye point can determine mesh or box.
+		GVREyePointeeHolder eyePointeeHolder = new GVREyePointeeHolder(
+                mGVRContext);
+        GVRMeshEyePointee eyePointee = new GVRMeshEyePointee(mGVRContext,
+                mMainObject.getRenderData().getMesh());
+        
+        // If want to use box type, like this.
+        GVRMeshEyePointee eyeRectPointee = new GVRMeshEyePointee(mGVRContext,
+        		mMainObject.getRenderData().getMesh().getBoundingBox());
+        
+        eyePointeeHolder.addPointee(eyePointee);
+        mMainObject.attachEyePointeeHolder(eyePointeeHolder);
+	}
+	
+	
+	
+	// 8. picking object.
+	private void checkEyePointee(){
```

Comment:
What is the point of this? We already have an eye picking sample that actually does something ...